sdk/resourcemanager/azure-resourcemanager-samples/src/main/java/com/azure/resourcemanager/appservice/samples/ManageLinuxWebAppSqlConnection.java

```diff
@@ -1,161 +1,161 @@
-                    .attach()
-                    .withAppSetting("ProjectNami.DBHost", server.fullyQualifiedDomainName())
-                    .withAppSetting("ProjectNami.DBName", db.name())
-                    .withAppSetting("ProjectNami.DBUser", admin)
-                    .withAppSetting("ProjectNami.DBPass", password)
-                    .create();
-
-            System.out.println("Created web app " + app.name());
-            Utils.print(app);
-
-            //============================================================
-            // Allow web app to access the SQL server
-
-            System.out.println("Allowing web app " + appName + " to access SQL server...");
-
-            SqlServer.Update update = server.update();
-            for (String ip : app.outboundIPAddresses()) {
-                update = update.defineFirewallRule("filewallRule1").withIpAddress(ip).attach();
-            }
-            server = update.apply();
-
-            System.out.println("Firewall rules added for web app " + appName);
-            Utils.print(server);
-
-            System.out.println("Your WordPress app is ready.");
-            System.out.println("Please navigate to http://" + appUrl + " to finish the GUI setup. Press enter to exit.");
-            System.in.read();
-
-            return true;
-        } finally {
-            try {
-                System.out.println("Deleting Resource Group: " + rgName);
-                azureResourceManager.resourceGroups().beginDeleteByName(rgName);
-                System.out.println("Deleted Resource Group: " + rgName);
-            } catch (NullPointerException npe) {
-                System.out.println("Did not create any resources in Azure. No clean up is necessary");
-            } catch (Exception g) {
-                g.printStackTrace();
-            }
-        }
-    }
-
-    /**
-     * Main entry point.
-     *
-     * @param args the parameters
-     */
-    public static void main(String[] args) {
-        try {
-
-            //=============================================================
-            // Authenticate
-
-            final AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
-            final TokenCredential credential = new DefaultAzureCredentialBuilder()
-                .authorityHost(profile.getEnvironment().getActiveDirectoryEndpoint())
-                .build();
-
-            AzureResourceManager azureResourceManager = AzureResourceManager
-                .configure()
-                .withLogLevel(HttpLogDetailLevel.BASIC)
-                .authenticate(credential, profile)
-                .withDefaultSubscription();
-
-            // Print selected subscription
-            System.out.println("Selected subscription: " + azureResourceManager.subscriptionId());
-
-            runSample(azureResourceManager);
-        } catch (Exception e) {
-            System.out.println(e.getMessage());
-            e.printStackTrace();
-        }
-    }
-}
+//
```

Comment:
Do we have upcoming cleanup PR's to delete these files rather than commenting out, especially since we are heading out for GA?

cc: @JonathanGiles @srnagar @weidongxu-microsoft

Reply:
The reason is that about 10 package get GAed, but the preview version of AzureResourceManager got about 20 packages.
Hence for a GA version of AzureResourceManager, the samples that using the non-GA package is commented out (e.g. ManageLinuxWebAppSqlConnection.java get commented out because sql package is non-GA, and it will not get published to Azure-Sample until sql package get GAed).
 
All of the commented-out sample would be put back when corresponding package get GAed (personally I hope it will be reverted sooner).

sample package will not be published to maven. It is for publish to repos on Azure-Sample.